tox.ini

```diff
@@ -51,4 +52,4 @@ commands =
     npm config set spin false
     npm install
     npm run lint
-    npm run package
+    npm run sign
```

Comment:
That, or just comment out this step for now and we can follow up in #447 over on CircleCI